app/models/student_profile.py

```diff
@@ -0,0 +1,29 @@
+from flask import current_app
+from flask_login import AnonymousUserMixin, UserMixin
+from itsdangerous import TimedJSONWebSignatureSerializer as Serializer
+from itsdangerous import BadSignature, SignatureExpired
+from werkzeug.security import check_password_hash, generate_password_hash
```

Comment:
Most of these import statements are for the User model. Generally you only need to include "from .. import db" when you're first creating a new model.